docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -124,7 +73,27 @@ An application name should make a valid sub-domain (\<=63 characters, no spaces,
 After Teleport is running, users can access the app at `app-name.proxy_public_addr.com`
 e.g. `grafana.teleport.example.com`. You can also override `public_addr` e.g
 `grafana.acme.com` if you configure the appropriate DNS entry to point to the
-Teleport proxy server.
+Teleport proxy server for self-hosted Teleport instances.
+
+<Admonition type="tip" title="Application Domain Names and TLS" scope="oss,enterprise" scopeOnly>
```

Comment:
Looking at this now, I think the body of the "Application name" section, including this `Admonition`, should appear where we first instruct the user to name their application, i.e., where we include the `tctl tokens add` command. Otherwise, a user will have already generated a token for a potentially misnamed application. 

Instead of making this a separate section, I would include the first two paragraphs of the section below the code snippet in "Generate a token," with a transition sentence that a reader should follow these instructions when naming their application. We can then refer to this name later when starting the Application Service.